playbooks/aws/openshift-cluster/tasks/launch_instances.yml

```diff
@@ -169,6 +177,7 @@
     ec2_private_ip_address: "{{ item.1.private_ip }}"
     ec2_ip_address: "{{ item.1.public_ip }}"
     openshift_node_labels: "{{ node_label }}"
+    openshift_schedulable: "{{ node_schedulable }}"
```

Comment:
Instead of setting openshift_schedulable here (and using the set_facts above, you'll probably want to re-add some logic back to openshift_facts to set the default.

Something like(in very rough psuedo code):
```
def set_node_schedulability(facts):
    if 'node' in facts:
        if 'schedulable' not in facts['node']:
            if 'master' in facts:
                facts['node']['schedulable'] = False
            else:
                facts['node']['schedulable'] = True
    return facts
```